extensions/arc/src/localizedConstants.ts

```diff
@@ -152,7 +162,10 @@ export const details = localize('arc.details', "Details");
 export const lastUpdated = localize('arc.lastUpdated', "Last updated");
 export const noExternalEndpoint = localize('arc.noExternalEndpoint', "No External Endpoint has been configured so this information isn't available.");
 export const podsReady = localize('arc.podsReady', "pods ready");
+export const connectToPostgresDescription = localize('arc.connectToPostgresDescription', "A connection to server is required to show and set database engine settings. Make sure to have installed PostgreSQL extension for Azure Data Studio.");
```

Comment:
`A connection to server is required to show and set database engine settings, which will require the PostgreSQL Extension to be installed.`